app/helpers/course_helper.rb

```diff
@@ -63,4 +63,15 @@ def course_applications_open(course)
   def course_send(course)
     course.attributes[:is_send?] == true ? 'Yes' : 'No'
   end
+
+  def course_length(course)
+    case course.attributes[:course_length]
+    when 'OneYear'
+      '1 year'
+    when 'TwoYears'
+      'Up to 2 years'
+    when 'Other'
+      'Other'
```

Comment:
Is this a reproduction of C#? If the value is "Other" it's meant to say the custom value the user entered.

![Screen Shot 2019-04-22 at 10 41 12](https://user-images.githubusercontent.com/319055/56494734-2c030b80-64eb-11e9-8868-b843188cf53a.png)

Reply:
Ah, good spot. Will check if we expose this on the backend and pull it through

Reply:
Done